Library/Formula/libqglviewer.rb

```diff
@@ -19,6 +19,12 @@ class Libqglviewer < Formula
 
   depends_on 'qt'
 
+  # This patches makes the package install QGLViewer.framework under
+  # #{prefix}/lib, where it will be picked by homebrew.
```

Comment:
audit is complaining about `#{prefix}/lib` which should be `#{lib}`. It's stupid because it's a comment but would you mind fixing? Ta.